app/views/users/update.html.erb

```diff
@@ -1,3 +1,11 @@
+<div class="pure-menu pure-menu-open pure-menu-horizontal">
+    <ul>
+        <li></li>
+        <li class="pure-menu-selected"></li>
+        <li></li>
+    </ul>
+</div>
+
```

Comment:
Is there a reason to have this menu div as the top banner instead of the echoThat branding? If not, I'd prefer to take the menu out.